src/channel/mpmc.rs

```diff
@@ -69,6 +69,35 @@ fn wakeup_last_receive_waiter(waiters: &mut LinkedList<RecvWaitQueueEntry>) {
     }
 }
 
+/// Conveys additionnal information regarding the status of a channel
+/// following a `close` operation.
+#[derive(Debug, Clone, Copy)]
+pub enum CloseStatus {
```

Comment:
Can you maybe add `PartialEq, Eq` for people who do not use the methods and do direct comparisons?

Reply:
Good point.